### PR TITLE
Feature/universal linking

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
This PR must be used in tandem with
https://github.com/helium/maker-starter-app/pull/10

### CODE REVIEW

File: ios/Helium/Info.plist [hunk 1]
```diff
@@ -1,117 +1,121 @@
...
+		</array>
+		<key>UISupportedInterfaceOrientations</key>
+		<array>
+			<string>UIInterfaceOrientationPortrait</string>
+		</array>
+		<key>UIViewControllerBasedStatusBarAppearance</key>
+		<false />
+		<key>LSApplicationQueriesSchemes</key>
+		<array>
+			<string>makerappscheme</string>
```
Comment (tyler-whitman) [inline ios/Helium/Info.plist:118]: So will each app need to be added as a scheme?

File: ios/Podfile.lock [hunk 2]
```diff
@@ -55,6 +55,8 @@ PODS:
     - ReactCommon/turbomodule/core (= 0.66.1)
   - fmt (6.2.1)
   - glog (0.3.5)
+  - helium-react-native-sdk (0.1.1):
```
Comment (tyler-whitman) [inline ios/Podfile.lock:58]: I dont think the latest pod got installed

File: src/features/txnDelegatation/SignHotspot.tsx [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+          finalTxn = stakingServerSignedTxn
+        }
+
+        responseParams.assertTxn = finalTxn
+      }
+
+      callback(responseParams)
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e)
```
Comment (tyler-whitman) [inline src/features/txnDelegatation/SignHotspot.tsx:108]: Do we want to log this in sentry?

File: src/features/txnDelegatation/SignHotspot.tsx [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+                variant="bold"
+                fontSize={20}
+                color="primaryBackground"
+                marginBottom="m"
+              >
+                {locationTxn.gain}
+              </Text>
+            </Box>
+          )}
+          {locationTxn?.elevation && (
```
Comment (tyler-whitman) [inline src/features/txnDelegatation/SignHotspot.tsx:192]: I think elevation can be 0, might want to check `locationTxn?.elevation !== undefined` instead

File: src/navigation/NavigationRoot.tsx [hunk 5]
```diff
@@ -33,6 +35,7 @@ const NavigationRoot = () => {
             component={Onboarding}
             options={{ gestureEnabled: false }}
           />
+          {/* TODO: Handle when someone tries to link an account or sign a txn and they aren't authenticated in the app */}
```
Comment (tyler-whitman) [inline src/navigation/NavigationRoot.tsx:38]: Good point, would this just go to some kind of error screen or maybe even just show a toast?

Comment (matthewcarlreetz) [reply src/navigation/NavigationRoot.tsx]: should be handled now

File: src/features/txnDelegatation/SignHotspot.tsx (hunk 3 above)
Comment (matthewcarlreetz) [reply src/features/txnDelegatation/SignHotspot.tsx:108]: that's a good idea

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**